src/ngResource/resource.js

```diff
@@ -443,10 +443,14 @@ angular.module('ngResource', ['ng']).
           });
 
           params = params || {};
-          forEach(self.urlParams, function(_, urlParam) {
+          forEach(self.urlParams, function(paramInfo, urlParam) {
             val = params.hasOwnProperty(urlParam) ? params[urlParam] : self.defaults[urlParam];
             if (angular.isDefined(val) && val !== null) {
-              encodedVal = encodeUriSegment(val);
+              if (paramInfo.isQueryParamiValue === true) {
```

Comment:
_isQueryParamiValue_  -->  _isQueryParamValue_

(It's a good idea to locally run the tests before pushing, to ensure nothing is (obviously) breaking :smiley:)